packages/rest/test/unit/coercion/paramStringToBoolean.unit.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {runTests} from './utils';
+
+describe('coerce param from string to boolean', () => {
+  /*tslint:disable:max-line-length*/
+  const testCases = [
+    ['false', {type: 'boolean'}, 'false', false, new Error().stack],
+    ['true', {type: 'boolean'}, 'true', true, new Error().stack],
+  ];
+
+  runTests(testCases);
```

Comment:
Ah, this is the design from strong-remoting that I am not very happy about. Because we invoke a single `runTests` function, all test cases are sharing the same stack and therefore you have to repeatedly call `new Error.stack` in test case definitions.

Here is what I think would work better - would you mind giving it a try?

```ts
describe('coerce param from string to boolean', () => {
  const spec = {type: 'boolean'};

  runTest(spec, 'false', false);
  runTest(spec, 'true', true);
});

function runTest(input, spec, expectedOutput) {
  const caller = new Error().stack.split(/\n/)[1];
  it(`converts ${input} to ${expectedOutput}`, async () => {
    await testCoercion({
       schema: spec,
       valueFromReq: input,
       expectedResult: expectedOutput,
       caller: caller,
     });
   });
}
```

This is just a quick mock-up to show what I mean, feel free to change it as you like.

Reply:
(I would also rename `runTest` to something like `test` or even `t`.)

Reply:
@bajtos thank you it works!